hie-compat/hie-compat.cabal

```diff
@@ -1,41 +1,59 @@
-cabal-version:       1.22
-name:                hie-compat
-version:             0.3.1.2
-synopsis:            HIE files for GHC 8.8 and other HIE file backports
-license:             Apache-2.0
+cabal-version: 1.22
+name: hie-compat
+version: 0.3.1.2
+synopsis: HIE files for GHC 8.8 and other HIE file backports
+license: Apache-2.0
 description:
   Backports for HIE files to GHC 8.8, along with a few other backports
   of HIE file related fixes for ghcide.
-
   THIS DOES NOT LET YOU READ HIE FILES WITH MISMATCHED VERSIONS OF GHC
-license-file:        LICENSE
-author:              Zubin Duggal
-maintainer:          zubin.duggal@gmail.com
-build-type:          Simple
-extra-source-files:  CHANGELOG.md README.md
-category:            Development
-homepage:            https://github.com/haskell/haskell-language-server/tree/master/hie-compat#readme
-bug-reports:         https://github.com/haskell/haskell-language-server/issues
+
+license-file: LICENSE
+author: Zubin Duggal
+maintainer: zubin.duggal@gmail.com
+build-type: Simple
+extra-source-files:
+  CHANGELOG.md
+  README.md
+
+category: Development
+homepage: https://github.com/haskell/haskell-language-server/tree/master/hie-compat#readme
+bug-reports: https://github.com/haskell/haskell-language-server/issues
 
 source-repository head
-    type:     git
-    location: https://github.com/haskell/haskell-language-server.git
+  type: git
+  location: https://github.com/haskell/haskell-language-server.git
 
 library
-  default-language:    GHC2021
+  default-language: GHC2021
+  build-depends:
+    array,
+    base <4.20,
+    bytestring,
+    containers,
+    directory,
+    filepath,
+    transformers
+
   build-depends:
-     base < 4.20, array, bytestring, containers, directory, filepath, transformers
-  build-depends: ghc >= 8.10, ghc-boot
-  ghc-options: -Wall -Wno-name-shadowing
+    ghc >=8.10,
+    ghc-boot
+
+  ghc-options:
+    -Wall
+    -Wno-name-shadowing
 
   exposed-modules:
     Compat.HieAst
     Compat.HieBin
-    Compat.HieTypes
     Compat.HieDebug
+    Compat.HieTypes
     Compat.HieUtils
 
-  if (impl(ghc >= 9.2) && impl(ghc < 9.3))
-    hs-source-dirs: src-ghc92 src-reexport-ghc9
+  if (impl(ghc >= 9.2)&& impl(ghc < 9.3))
```

Comment:
Hmm this formatting is no good. There should be a space before the `&&`. I made an issue for this: https://github.com/tfausak/cabal-gild/issues/72

Reply:
This has been fixed in version 1.3.1.3: https://github.com/tfausak/cabal-gild/releases/tag/1.3.1.3